packages/terra-form-checkbox/src/CheckboxField.jsx

```diff
@@ -113,7 +113,12 @@ const CheckboxField = (props, { intl }) => {
     <legend id={legendAriaDescriptionId} className={cx(['legend-group', { 'legend-group-hidden': isLegendHidden }])}>
       <div {...legendAttrs} className={legendClassNames}>
         {isInvalid && <span className={cx('error-icon')} />}
-        {required && (isInvalid || !hideRequired) && <span className={cx('required')}>*</span>}
+        {required && (isInvalid || !hideRequired) && (
+          <React.Fragment>
+            <div aria-hidden="true" className={cx('required')}>*</div>
+            <span className={cx('visually-hidden-text')}>Required </span>
```

Comment:
Waiting for translations for this string

Reply:
Added in https://github.com/cerner/terra-core/pull/2398/commits/5b2f66f50e88c186e8d826512d3ff2f3d974991c